benchmark_runner/common/clouds/BM/bare_metal_exceptions.py

```diff
@@ -0,0 +1,32 @@
+
+class BMError(Exception):
+    """ Base class for all Bare metal error classes.
+        All exceptions raised by the benchmark runner library should inherit from this class. """
+    pass
+
+
+class MissingMasterNodes(BMError):
+    """
+    This class exception is for missing master nodes
+    """
+    def __init__(self):
+        self.message = 'There are missing master nodes'
+        super(MissingMasterNodes, self).__init__(self.message)
+
+
+class MissingWorkerNodes(BMError):
+    """
+    This class exception is for missing worker nodes
+    """
+    def __init__(self):
+        self.message = 'There are missing worker nodes'
```

Comment:
Likewise.

Reply:
Its not so important which master node is missing, cluster is corrupted and the user should investigate it by himself

Reply:
Same, please list what node(s) are missing.

Reply:
Fixed